cachecontrol/caches/file_cache.py

```diff
@@ -62,12 +69,12 @@ class _FileCacheMixin:
 
     def __init__(
         self,
-        directory,
-        forever=False,
-        filemode=0o0600,
-        dirmode=0o0700,
-        use_dir_lock=None,
-        lock_class=None,
+        directory: str,
+        forever: bool = False,
+        filemode: int = 0o0600,
+        dirmode: int = 0o0700,
+        use_dir_lock: Optional[bool] = None,
```

Comment:
slightly curious API here, I've annotated it as it is.  

Suggest that `use_dir_lock: bool = False` might have done, though.

Reply:
removed in 0.12.12 anyway